hieradata/production.yaml

```diff
@@ -68,6 +68,8 @@ base::supported_kernel::enabled: false
 
 environment_ip_prefix: '10.3'
 
+govuk::apps::content_audit_tool::ensure: 'absent'
```

Comment:
Are you planning to change this to present at some point?

Reply:
@cbaines yep. We need to coordinate with a dept on when we can do the switchover. I thought it was a good idea to use this flag for now and not worry about the production deployment just yet.

Reply:
Ok, that sounds fine :)